packages/storage/amplify_storage_s3_dart/test/path_resolver/path_resolver_test.dart

```diff
@@ -7,6 +7,13 @@ import 'package:test/test.dart';
 
 class MockTokenIdentityAmplifyAuthProvider
     implements TokenIdentityAmplifyAuthProvider {
+  MockTokenIdentityAmplifyAuthProvider({
+    this.getIdException,
+  });
+
+  Exception? getIdException;
+  int getIdCount = 0;
```

Comment:
Q: Is this being used?

Reply:
Yes - in the test "should resolve multiple paths with identity Id". It is used to confirm getId is only called once when resolving multiple paths.